docs/getting-started/commands_reference.md

```diff
@@ -72,133 +66,134 @@ The commands a project supports are specified on the framework side. If you want
-
-A parameterised run is best used for dynamic parameters, i.e. running the same pipeline with different inputs, for static parameters that do not change we recommend following the [Kedro project setup methodology](../configure/parameters.md).
-
-### Deploy the project
-
-The following packages your application as one `.whl` file within the `dist/` folder of your project. It packages the project configuration separately in a `tar.gz` file:
-
-```bash
-kedro package
-```
-
-See [the Python documentation for further information about packaging](https://packaging.python.org/overview/).
-
-### Project quality
-
-### Project development
-
-#### Modular pipelines
-
-##### Create a new [modular pipeline](../build/modular_pipelines.md) in your project
-
-```bash
-kedro pipeline create <pipeline_name>
-```
-
-##### Delete a modular pipeline
-The following command deletes all the files related to a modular pipeline in your Kedro project.
-
-```bash
-kedro pipeline delete <pipeline_name>
-```
-
-#### Registered pipelines
-
-##### Describe a registered pipeline
-
-```bash
-kedro registry describe <pipeline_name>
-```
-The output includes all the nodes in the pipeline. If no pipeline name is provided, this command returns all nodes in the `__default__` pipeline.
-
-##### List all registered pipelines in your project
-
-```bash
-kedro registry list
-```
-
-#### Notebooks
-
-To start a Jupyter Notebook:
-
-```bash
-kedro jupyter notebook
-```
-
-To start JupyterLab:
-
-```bash
-kedro jupyter lab
-```
-
-To start an IPython shell:
-
-```bash
-kedro ipython
-```
+!!! note
+    All project related CLI commands should be run from the project’s root directory.
 
-The [Kedro IPython extension](../integrations-and-plugins/notebooks_and_ipython/kedro_and_notebooks.md#what-does-kedro-jupyter-notebook-do) makes the following variables available in your IPython or Jupyter session:
+Kedro's command line interface (CLI) allows you to associate a set of commands and dependencies with a target, which you can then execute from inside the project directory.
 
-* `catalog` (type [kedro.io.DataCatalog][]): [Data Catalog](../catalog-data/data_catalog.md) instance that contains all defined datasets; this is a shortcut for `context.catalog`
-* `context` (type [kedro.framework.context.KedroContext][]): Kedro project context that provides access to Kedro's library components
-* `pipelines` (type `dict[str, Pipeline]`): Pipelines defined in your [pipeline registry](../build/run_a_pipeline.md#run-a-pipeline-by-name)
-* `session` (type [kedro.framework.session.session.KedroSession][]): [Kedro session](../extend/session.md) that orchestrates a pipeline run
+The commands a project supports are specified on the framework side. If you want to customise any of the Kedro commands you can do this either by adding a file called `cli.py` or by injecting commands into it via the [`plugin` framework](../extend/plugins.md). Find the template for the `cli.py` file below.
```

Comment:
I find this is less clear than what we had before https://docs.kedro.org/en/0.19.14/extend_kedro/common_use_cases.html#use-case-3-how-to-add-or-modify-cli-commands

"framework side" probably will not make sense to users. I agree that the code block needs to be here